packages/terra-application-header-layout/translations/de.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
We need `ar` translatios as well

Reply:
https://github.com/cerner/terra-framework/pull/301/commits/932608d8f23365316a8f4bf72b180e68907596c3